traits/Sluggable.php

```diff
@@ -0,0 +1,113 @@
+<?php namespace RainLab\Translate\Traits;
+
+use October\Rain\Support\Str;
```

Comment:
@mariavilaro This can be shortened to just use the facade :)
```suggestion
use Str;
```